src/messages/responses/cache-set-remove-elements.ts

```diff
@@ -1 +1,25 @@
-export * from './response-simple-success';
+import {ResponseBase, ResponseError, ResponseSuccess} from './response-base';
+import {SdkError} from '../../errors/errors';
+import * as CacheSetRemoveElement from './cache-set-remove-element';
+
+export abstract class Response extends ResponseBase {
+  abstract toSingularResponse(): CacheSetRemoveElement.Response;
+}
+
+class _Success extends Response {
+  toSingularResponse(): CacheSetRemoveElement.Response {
+    return new CacheSetRemoveElement.Success();
+  }
+}
```

Comment:
@schwern curious to hear if you have an opinion on how to extend the `./response-simple-success` construct to include  `toSingularResponse()`

Reply:
@virratanasangpunth Something like this maybe?

```
import { default as SimpleSuccess } from './response-simple-success'

export class Success extends SimpleSuccess.Success {
  toSingularResponse(): CacheSetRemoveElement.Response {
    return new CacheSetRemoveElement.Success();
  }
}
```

Reply:
@virratanasangpunth This works.

```
import * as SimpleSuccess from './response-simple-success';
import * as CacheSetAddElement from './cache-set-add-element';

export abstract class Response extends SimpleSuccess.Response {
  abstract toSingularResponse(): CacheSetAddElement.Response;
}

export class Success extends SimpleSuccess.Success {
  toSingularResponse(): CacheSetAddElement.Response {
    return new CacheSetAddElement.Success();
  }
}

export class Error extends SimpleSuccess.Error {
  toSingularResponse(): CacheSetAddElement.Response {
    return new CacheSetAddElement.Error(this._innerException);
  }
}
```